amivapi/settings.py

```diff
@@ -116,3 +116,37 @@
     'mtec',
     'gess'
 ]
+
+# Config for swagger (/docs)
+# TODO: should be linked to the setup.py information in the package
+SWAGGER_INFO = {
+    'title': 'AMIVAPI',
+    'version': '0.1-dev',
+    'description': "The REST API behind most of AMIV's web services.",
+    'termsOfService': 'todo',
+    'contact': {
+        'name': 'AMIV an der ETH',
+        'url': 'http://amiv.ch/'
+    },
+    'license': {
+        'name': 'GNU',
+        'url': 'https://github.com/amiv-eth/amivapi/blob/master/LICENSE',
+    }
+}
+ENABLE_HOOK_DESCRIPTION = True
+HIDE_HOOK_FUNCTIONS = ['authenticate',
+                       'check_if_admin',
+                       'abort_if_not_public',
+                       'add_lookup_filter',
+                       'check_resource_write_permission',
+                       'check_item_write_permission',
+                       'add_permitted_methods_after_insert'
+                       'add_permitted_methods_after_fetch_resource',
+                       'add_permitted_methods_after_fetch_item',
+                       'add_permitted_methods_after_fetch_resource',
+                       'add_permitted_methods_for_home',
+                       'add_permitted_methods_after_update']
+X_DOMAINS = ['http://127.0.0.1:5000',  # The domain where Swagger UI is running
```

Comment:
This IP should not be hardcoded, this breaks the documentation in many setups. The address should be available via flask. (maybe SERVER_NAME config value?)